pages/profile/[id].js

```diff
@@ -0,0 +1,50 @@
+import React, {useEffect, useState} from 'react'  
+import BasicInfo from '../../components/Profile/BasicInfo'
+import VotingPosition from '../../components/Profile/VotingPosition.js'
+import * as member from '../api/propublica/member'
+
+//need to get member Id from the route props
+
+export async function getServerSideProps(context){
+    const { id } = context.params;
+
+    return {
+        props: {
+            id,
+        },
+    };
+}
+
+const Profile = ({ id }) => {
+    const [votingPositions, setVotingPositions] = useState([])
+    const [memberInfo, setMemberInfo] = useState()
+
+    useEffect(() => {
+        const memberInformation = member.getMemberById(id)
+        const votingPos = member.getVotingPositions(id)
+        votingPos.then(({results}) => setVotingPositions(results[0].votes));
+        memberInformation.then(({results}) => setMemberInfo(results[0]))
+    }, [])
+
+    const renderBasicInfo = () => {
+        if(memberInfo){
+            return <BasicInfo data={memberInfo}/>
+        }else{
+            return <div></div>
+        }
```

Comment:
You can return the empty the null state directly 
```suggestion
        if(memberInfo){
          return <BasicInfo data={memberInfo}/>
        }
       return <div></div>
```